### DISCUSSION TITLE
Many serious bugs in map.sh - Please update.

### DISCUSSION THREAD

**Discussion by cmspam** [fakemanhk/openwrt-jp-ipoe#13] [General]
It's my fault, but I was very sloppy when putting together the map.sh which is currently listed, and there are cases where traffic is not routed correctly.

Everything has been rewritten and features added, so those who have been using the old version (anything before March 5, just now), please update to ensure it works.

You can find the file in the issue I opened here:
https://github.com/fakemanhk/openwrt-jp-ipoe/issues/12

I expect @fakemanhk will update the main page with the new script soon.

My apologies for releasing something incomplete before.

**Comment by fakemanhk** [comment]
Many thanks! My plala (which is same as OCN) has worked with this new version very well.

**Reply by cmspam** [reply]
That's good news. I believe it will work for anything the original map.sh script worked for, at this point.
